test/secret/secret_test.go

```diff
@@ -120,6 +120,58 @@ var _ = Describe("secret test", func() {
 
 			// Verify that Pass4SymmKey is updated
 			testenv.VerifySecretsUpdatedOnPod(deployment, testenvInstance, verificationPods, testenv.SecretObject["Pass4SymmKey"], testenv.DecodeBase64(secretObj.Data.Pass4SymmKey))
+
+			// Delete secret key
+			testenv.UpdateSecret(deployment, testenvInstance.GetName(), secretObj, true)
+
+			// Ensure standalone is updating
+			testenv.VerifyStandalonePhase(deployment, testenvInstance, deployment.GetName(), splcommon.PhaseUpdating)
+
+			// Wait for License Master to be in READY status
+			testenv.LicenseMasterReady(deployment, testenvInstance)
+
+			// Wait for Standalone to be in READY status
+			testenv.StandaloneReady(deployment, deployment.GetName(), standalone, testenvInstance)
+
+			// Verify MC Pod is Ready
+			testenv.MCPodReady(testenvInstance.GetName(), deployment)
+
+			// Once system is up after update check each pod for secret key update
```

Comment:
Change comment to say
`Once Pods are check each versioned secret for updated secret keys`

Reply:
updated